### DISCUSSION TITLE
AI Search

### DISCUSSION THREAD

**Discussion by Shubham-EV** [fuma-nama/fumadocs#948] [Q&A]
![image](https://github.com/user-attachments/assets/51e3e57e-0f22-4988-8a1a-40d880aad4c9)

How to use and customize this AI feature for my own docs website?

And I have also observed some pages have extra space at bottom for eg - [Layouts](https://fumadocs-dev.vercel.app/docs/ui/blocks)

![image](https://github.com/user-attachments/assets/dbe57647-dbeb-424d-ab4d-f7e66ff16d16)

**Comment by fuma-nama** [comment]
> How to use and customize this AI feature for my own docs website?

This is not meant to be a feature, you need to configure it manually and choose an AI solution like inkeep or Orama Cloud.

> And I have also observed some pages have extra space at bottom for eg

Add min height as `100vh` and `display: flex; flex-direction: column;` to your body

**Comment by khansais557-dotcom** [comment]
Mujhe ek video Banakar ke do jismein Modi a Rahe hamesha chai chahie

**Comment by khansais557-dotcom** [comment]
Mujhe ek video Banakar ke do jismein Modi a Rahe hamesha chai chahie

**Comment by khansais557-dotcom** [comment]
Mujhe ek video Banakar ke do jismein Modi a Rahe hamesha chai chahie

**Comment by khansais557-dotcom** [comment]
Mujhe ek video Banakar ke do jismein Modi a Rahe hamesha chai chahie
